src/root.component.tsx

```diff
@@ -8,6 +8,76 @@ defineConfigSchema("@openmrs/esm-home", {
   buttons: {
     enabled: {
       default: false
+    },
+    list: {
+      arrayElements: {
+        label: { validators: [validators.isString] },
+        link: {
+          spa: { validators: [validators.isBoolean] },
+          url: { validators: [validators.isString] }
+        }
+      },
+      default: [
+        {
+          label: "Active Visits",
+          link: {
+            spa: false,
+            url: "/openmrs/coreapps/activeVisits.page?app=coreapps.activeVisits"
+          }
+        },
+        {
+          label: "Register Patient",
+          link: {
+            spa: false,
+            url:
+              "/openmrs/registrationapp/registerPatient.page?appId=referenceapplication.registrationapp.registerPatient"
+          }
+        },
+        {
+          label: "Capture Vitals",
+          link: {
+            spa: false,
```

Comment:
why does `spa` default to `false`? don't we already have a mechanism to capture the vitals?

Reply:
This is a specific "app" for rapidly collecting vitals from lots of different patients (e.g. that would be used by a nurse doing triage). It doesn't yet have a SPA equivalent.

Also, I don't really expect anyone to use the default home button configuration. I think its purpose should be seen as showing the implementer what capabilities are available. I think we should feel pretty free to add stuff, as new MFEs get developed. (My mind is open to being changed on all of this).

Reply:
sounds good 👍